spec.html

```diff
@@ -32086,27 +32086,128 @@ <h1>Week Day</h1>
+          The elements are sorted by ascending numerical value.
+          When the input represents a local time skipped because of a positive time zone transition (e.g. when daylight saving time begins or the time zone offset is increased due to a time zone rule change), the returned List will be empty.
+          Otherwise, the returned List will have one element.
+        </p>
+        <p>The minimum implementation of GetNamedTimeZoneEpochNanoseconds for ECMAScript implementations that do not include local political rules for any time zones performs the following steps when called:</p>
+        <emu-alg>
+          1. Assert: _timeZoneIdentifier_ is *"UTC"*.
+          1. Let _epochNanoseconds_ be GetEpochNanosecondsFromISOParts(_year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_).
+          1. Return &laquo; _epochNanoseconds_ &raquo;.
+        </emu-alg>
         <emu-note>
           <p>It is recommended that implementations use the time zone information of the IANA Time Zone Database <a href="https://www.iana.org/time-zones/">https://www.iana.org/time-zones/</a>.</p>
-          <p>1:30 AM on 5 November 2017 in America/New_York is repeated twice (fall backward), but it must be interpreted as 1:30 AM UTC-04 instead of 1:30 AM UTC-05. LocalTZA(TimeClip(MakeDate(MakeDay(2017, 10, 5), MakeTime(1, 30, 0, 0))), false) is <emu-eqn>-4 &times; msPerHour</emu-eqn>.</p>
-          <p>2:30 AM on 12 March 2017 in America/New_York does not exist, but it must be interpreted as 2:30 AM UTC-05 (equivalent to 3:30 AM UTC-04). LocalTZA(TimeClip(MakeDate(MakeDay(2017, 2, 12), MakeTime(2, 30, 0, 0))), false) is <emu-eqn>-5 &times; msPerHour</emu-eqn>.</p>
-          <p>Local time zone offset values may be positive <i>or</i> negative.</p>
+          <p>1:30 AM on 5 November 2017 in America/New_York is repeated twice, so GetNamedTimeZoneEpochNanoseconds(*"America/New_York"*, 2017, 11, 5, 1, 30, 0, 0, 0, 0) would return a List of length 2 in which the first element represents 05:30 UTC (corresponding with 01:30 US Eastern Daylight Time at UTC offset -04:00) and the second element represents 06:30 UTC (corresponding with 01:30 US Eastern Standard Time at UTC offset -05:00).</p>
+          <p>2:30 AM on 12 March 2017 in America/New_York does not exist, so GetNamedTimeZoneEpochNanoseconds(*"America/New_York"*, 2017, 3, 12, 2, 30, 0, 0, 0, 0) would return an empty List.</p>
+        </emu-note>
+      </emu-clause>
+
+      <emu-clause id="sec-getnamedtimezoneoffsetnanoseconds" oldids="sec-local-time-zone-adjustment" type="implementation-defined abstract operation">
+        <h1>
+          GetNamedTimeZoneOffsetNanoseconds (
+            _epochNanoseconds_: a BigInt,
+            _timeZoneIdentifier_: a String,
```

Comment:
Would it make more sense to swap the argument order, aligning with the preceding operation?
```suggestion
            _timeZoneIdentifier_: a String,
            _epochNanoseconds_: a BigInt,
```